agent/requirements.txt

```diff
@@ -1,15 +1,9 @@
 ansible
-boto3
 bottle
 cffi
 click
 colorlog
-elasticsearch1==1.10.0
-flask
-flask-restful
-gunicorn
 humanize
```

Comment:
Are we using `humanize` in the agent? I thought Nikhil just added it for the server?

Reply:
I'll remove this.

Reply:
Removed.